Firmware/RTK_Surveyor/System.ino

```diff
@@ -102,24 +156,29 @@ bool configureUbloxModule()
 
   if (response == false)
     systemPrintln("Module failed config block 0");
+  else
+    log_d("Module config block 0 complete");
   response = true; //Reset
 
   //Enable the constellations the user has set
   response &= setConstellations(true); //19 messages. Send newCfg or sendCfg with value set
   if (response == false)
     systemPrintln("Module failed config block 1");
+  else
+    log_d("Module config block 1 complete");
   response = true; //Reset
 
   //Make sure the appropriate messages are enabled
   response &= setMessages(); //73 messages. Does a complete open/closed val set
   if (response == false)
     systemPrintln("Module failed config block 2");
+  else
+    log_d("Module config block 2 complete");
```

Comment:
I recommend remove. I used the same thing during initial dev, but removed once proven.

Reply:
Indeed. I was still tracking down my Key ID UART1 -> SPI offset number error when I added this. It can be deleted now.